desub/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "desub"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+desub_legacy = { path = "../core/", package = "desub-core" }
+core_v14 = { path = "../core_v14/", package = "core_v14" }
```

Comment:
I wonder whether we should consistify these names to, maybe:

```
core_legacy
core_v14
```

or something (just so it's obvious that they are closely related when using).

Reply:
now that we've agreed on names, `desub-current` and `desub-legacy` I geuss would be the most future proof choices :)